### ISSUE TITLE
Filter in Edit user permission view is confusing 

### ISSUE THREAD

**Issue by Findus23** [matomo-org/matomo#13698]
This is a pure UX issue.

If you create a new user and want to give them read access to all websites, you will end up in this menu.
![https://forum.matomo.org/uploads/db1181/original/2X/d/d58d0be6651db6d9816e5ec39a6140d14433afe4.png](https://forum.matomo.org/uploads/db1181/original/2X/d/d58d0be6651db6d9816e5ec39a6140d14433afe4.png)

But while it seems clear that ticking the top tick selects all website, it is not obvious how you can get all websites to show up here.

![https://forum.matomo.org/uploads/db1181/original/2X/8/817a751cf5f5c9d81cf2a9da9fca85ca3f3d0e6b.png](https://forum.matomo.org/uploads/db1181/original/2X/8/817a751cf5f5c9d81cf2a9da9fca85ca3f3d0e6b.png)

The right selection has no label and the "show all" selection is the blank one. 
In addition the textbox to the left explains what it does, but in an nearly unreadable font color.

More information: https://forum.matomo.org/t/add-all-sites-to-user-permission/30407
